data/blog/0xB10C-receives-lts-grant.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Long-Term Support For 0xB10C"
+date: '2024-02-02'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['cfunk']
+images: ['/static/images/blog/23-0xB10C.jpg']
+summary: "OpenSats Welcomes 0xB10C as an LTS Grantee."
+---
+
+OpenSats is thrilled to introduce our latest LTS grantee:
+[0xB10C](https://github.com/0xB10C). He builds Bitcoin network monitoring tools,
+notably the Bitcoin Core tracing framework, providing detailed insights into
+network activities.
+
+His [miningpool-observer](https://miningpool.observer/) and
+[peer-observer](https://github.com/0xB10C/peer-observer) projects played a
+pivotal role in identifying and testing a fix for an undisclosed P2P network
+problem in the spring of 2023.
+
+Our grant will further assist him in reviewing and testing Bitcoin Core PRs from
+a data-driven approach, ensuring informed decision-making and validating
+proposed changes with real-world testing. Additionally, it will support the
+publication of a v1.0 for projects like [fork-observer](https://fork.observer/)
```

Comment:
while getting these projects to a "v1.0" this year is a goal,   it might sound better to just say "Additionally, it will support the development of projects like ...".